src/ui/DynamicHierarchicalFacetSearch/DynamicHierarchicalFacetSearchValueRenderer.ts

```diff
@@ -0,0 +1,73 @@
+import { IDynamicHierarchicalFacet } from '../DynamicHierarchicalFacet/IDynamicHierarchicalFacet';
+import { DynamicHierarchicalFacetSearchValue } from './DynamicHierarchicalFacetSearchValue';
+import { $$ } from '../../utils/Dom';
+
+const PATH_START_LENGTH = 1;
+const PATH_END_LENGTH = 2;
+const PATH_SEPARATOR = '/';
```

Comment:
This can be configured as an option, no ? 🤔 Meaning we can't assume `/`

Reply:
That looks like it's for display purposes, following the mockups, it shouldn't really be configurable

Reply:
Yup, it's only for display purposes.